server/auth/login-gov.es6

```diff
@@ -0,0 +1,88 @@
+let passport = require('passport');
+let vcapServices = require('../vcap-services.es6');
+let Issuer = require('openid-client').Issuer;
+let Strategy = require('openid-client').Strategy;
+
+let router = require('express').Router();
+
+let loginGov = {};
+
+let basicAuthOptions = {
+  headers: {
+    Host: 'idp.int.login.gov',
+    Authorization:
+      'Basic ' +
+      new Buffer(vcapServices.loginGovIdpUsername + ':' + vcapServices.loginGovIdpPassword).toString('base64')
+  }
+};
+
+loginGov.setup = () => {
+  console.log('------------ in loginGov.setup ', vcapServices.loginGovIdpUsername, vcapServices.loginGovIdpPassword);
```

Comment:
In particular this is a good example, I know this was for testing but its not really complete software with these debugging console.log statements.